alphadia/workflow/optimization.py

```diff
@@ -400,13 +422,63 @@ def __init__(
         self.estimator_group_name = "precursor"
         self.estimator_name = "rt"
         self.feature_name = "precursor_proportion_detected"
+        self.maximal_decrease = workflow.config["optimization"]["rt_error"][
+            "maximal_decrease"
+        ]
+        self.minimum_proportion_of_maximum = workflow.config["optimization"][
+            "rt_error"
+        ]["minimum_proportion_of_maximum"]
         super().__init__(initial_parameter, workflow, reporter)
 
     def _get_feature_value(
         self, precursors_df: pd.DataFrame, fragments_df: pd.DataFrame
     ):
         return len(precursors_df) / self.workflow.optlock.total_elution_groups
 
+    def _check_convergence(self):
```

Comment:
(nit) from a method that start with `check_` I would always assume that something is raised if the check fails.. 
`_has_converged()`?

Reply:
That's already the name of the attribute set by a conditional if self._check_convergence() is True. The easiest thing might be to make just_converged a property with the current definition of self._check_convergence(), and then we could dispense with the function. This is probably another PR though.